scripts/piaware_config.py

```diff
@@ -210,6 +214,13 @@ def validate_uuid(self, val: str, version=4) -> bool:
     def validate_gain(self, val: str) -> bool:
         return self.validate_double(val) or (isinstance(val, str) and val == "max")
 
+    def check_if_file_in_ignore_list(self, key, file_name) -> bool:
```

Comment:
If you defaulted `ignore_list` to `[]` then this function becomes just `return (file_name in self.settings[key].ignore_list)` which is brief enough you may not even need a function for it.